example_scripts/gen_ta_dictionary.py

```diff
@@ -0,0 +1,112 @@
+import re, json
```

Comment:
Would be good to add header plus Apache 2.0 licence wording.

Reply:
Done